file.go

```diff
@@ -25,8 +25,10 @@ type textArea struct {
 	InjectTag  string
 }
 
-func parseFile(inputPath string, xxxSkip []string) (areas []textArea, err error) {
-	log.Printf("parsing file %q for inject tag comments", inputPath)
+func parseFile(inputPath string, xxxSkip []string, verbose bool) (areas []textArea, err error) {
+	if verbose {
+		log.Printf("parsing file %q for inject tag comments", inputPath)
```

Comment:
could you make `if verbose { log.Printf }` to a function like `logVerbose` or something?